web/src/components/Project/Project.jsx

```diff
@@ -62,7 +62,7 @@ export const Project = ({ data, onProjectChange }) => {
     <div className={styles.container}>
       <form className={styles.form}>
         <h1>PROJECTS</h1>
-        <p>Tell us about a project you’ve worked on.</p>
+        <h2>Tell us about a project you’ve worked on?</h2>
```

Comment:
The question mark here doesnt match the full stop from the Education component. These should all align